src/servicesHost.ts

```diff
@@ -852,6 +852,14 @@ export function makeSolutionBuilderHost(
   function getInputFileNameFromOutput(
     outputFileName: string
   ): string | true | undefined {
+    // Unless we explicitly want to compile files in node_modules, exclude them from lookups
+    if (
```

Comment:
As @andrewbranch i am not familiar with monorepo projects out there that this will impact. I think somewhere we are missing the cache and that would be better solution in my opinion